02_calculator/calculator.rb

```diff
@@ -1 +1,32 @@
 #write your code here
+def add(a, b)
+    return a + b
```

Comment:
Nit: Ruby convention omits the `return` if it's the last line of the method, so that it's an implicit return.

```suggestion
    a + b
```